.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       inputs:
         packageSpecs: 'python=3.6 conda-build>=3.18 conda!=4.6.1,<4.7.11a0 conda-forge::conda-forge-ci-setup=2 networkx=2.3 conda-forge-pinning' # Optional
         installOptions: "-c conda-forge"
-        updateConda: false
+        updateConda: true
```

Comment:
We should not change files outside of a recipe in a recipe PR.  CC @beckermr

Reply:
This was me @scopatz. I can put the change directly on master, but then we will have to rebase this PR or merge master into it.

Reply:
see here: https://github.com/conda-forge/staged-recipes/pull/11047

Reply:
Yep, that's fine.  That's what we normally do :wink: We mostly don't want to have recipe PRs affect each other